src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -397,6 +397,19 @@ message BundlingSettingsProto {
 
   // The maximum size of the request that could be accepted by server.
   uint32 request_byte_limit = 5;
+
+  uint32 flow_control_element_limit = 6;
+
+  uint32 flow_control_byte_limit = 7;
+
+  FlowControlLimitExceededBehavior flow_control_limit_exceeded_behavior = 8;
+}
+
+enum FlowControlLimitExceededBehavior {
```

Comment:
Can we name this `FlowControlLimitExceededBehaviorProto`? Most types in this file has "Proto" suffix to distinguish it from similar view classes.